{{cookiecutter.app_name}}/Makefile

```
@@ -28,6 +28,10 @@ endif
 migration/status:
 	goose -dir database/migrations -table "migration_versions" postgres "$(DATABASE_URL)" status
 
+{% if cookiecutter._api_variant == "yes" %}doc/generate:
```

How about changing the alignment here too:

```suggestion
{%- if cookiecutter._api_variant == "yes" %}
doc/generate:
```

---

@olivierobert I tried with this before, but it will remove the break line above it (line 30). The Makefile will be this:
```makefile
migration/status:
	goose -dir database/migrations -table "migration_versions" postgres "$(DATABASE_URL)" status
doc/generate:
	npm run build:docs
```

So I prefer to not make this change